Source/WebCore/platform/mediastream/gstreamer/GStreamerMediaStreamSource.cpp

```diff
@@ -124,8 +124,13 @@ class InternalSource final : public MediaStreamTrackPrivate::Observer,
         : m_parent(parent)
         , m_track(track)
         , m_padName(padName)
+#if USE(GSTREAMER_WEBRTC)
```

Comment:
This is incorrect... a non-webrtc mediastream can also be consumed by a player.

Reply:
Hm, I didn't make any behavior changes here: the variable really was unused. Do you want to report a bug to fix this?

Reply:
nevermind